CHANGELOG.md

```diff
@@ -7,12 +7,30 @@ and this project uses [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 
 ## [Unreleased]
 
+## [v0.13.0] - 2025-01-28
+
+### Changed
+- Code of conduct: "per NASA direction received on 1/22/2025, per Executive Order on 1/20/2025" [see more](https://en.wikipedia.org/wiki/Nineteen_Eighty-Four)
```

Comment:
As much as I like this, I think we shouldn't mention it because (1) risk of impact to our community if someone decides to punish us for this speech; (2) it has no impact on an end-user who installs the software with `pip` or `conda`.

The changelog isn't for contributor-facing stuff, or it gets too noisy and confusing for end-users.